app/assets/stylesheets/_category.scss

```diff
@@ -5,6 +7,17 @@
   margin-left: 15px;
   margin-right: 15px;
 
+  li{
+    @include border-radius(5px);
+    border:1px solid white;
+    color:white;
```

Comment:
Color `white` should be written in hexadecimal form as `#ffffff`<br>Color literals like `white` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Colon after property should be followed by one space